package.json

```diff
@@ -44,7 +44,7 @@
     "@endpass/class": "^0.20.1",
```

Comment:
and where new fonts declared? can't see any declaration for default texts.

Reply:
There used two variables: $headings-font-family (which was changed), using for headings, and `--endpass-ui-font-family-primary`, which inherited from `@endpass/ui`. So no another places for font-family here. When you will approve, you can check real font-family of text via devtools.